index.d.ts

```diff
@@ -584,20 +603,26 @@ declare global {
   type StrokeJoin = "MITER" | "BEVEL" | "ROUND"
   type HandleMirroring = "NONE" | "ANGLE" | "ANGLE_AND_LENGTH"
 
-  interface GeometryMixin {
-    fills: ReadonlyArray<Paint> | PluginAPI['mixed']
+  interface MinimalStrokesMixin {
     strokes: ReadonlyArray<Paint>
+    strokeStyleId: string
     strokeWeight: number
-    strokeMiterLimit: number
-    strokeAlign: "CENTER" | "INSIDE" | "OUTSIDE"
-    strokeCap: StrokeCap | PluginAPI['mixed']
     strokeJoin: StrokeJoin | PluginAPI['mixed']
+    strokeAlign: "CENTER" | "INSIDE" | "OUTSIDE"
     dashPattern: ReadonlyArray<number>
+  }
+  
+  interface MinimalFillsMixin {
+    fills: ReadonlyArray<Paint> | PluginAPI['mixed']
     fillStyleId: string | PluginAPI['mixed']
-    strokeStyleId: string
+  }
+  interface GeometryMixin extends MinimalStrokesMixin, MinimalStrokesMixin {
```

Comment:
One of these is meant to be `MinimalFillsMixin` right?

Reply:
Yep, fixed